src/platforms/javascript/common/install/loader.mdx

```diff
@@ -84,42 +84,71 @@ You can configure the release by adding the following to your page:
 
 ```html
+<script>
+  // Configure sentryOnLoad before adding the Loader Script
+  window.sentryOnLoad = function () {
+    Sentry.init({
+      // add custom config here
+    });
+  };
+</script>
+
 <script
   src="https://js.sentry-cdn.com/___PUBLIC_KEY___.min.js"
   crossorigin="anonymous"
 ></script>
-<script>
-  // Check for existence of Sentry in case Ad-blockers block the Sentry Loader Script
-  window.Sentry && Sentry.onLoad(function() { ... });
-</script>
 ```
 
-`Sentry.onLoad` is a function that only the loader provides, and - as the name suggests - it sets a function to be run once the full SDK has been loaded. In that function, you can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init` call doesn't need to include your DSN, since it's already been set. We recommend checking if `Sentry` exists before calling `Sentry.onLoad` in case the loader script failed to load.
+Inside of the `window.sentryOnLoad` function, you can configure a custom `Sentry.init()` call. You can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init()` call doesn't need to include your DSN, since it's already been set. Inside of this function, the full Sentry SDK is guaranteed to be loaded & available.
 
 ```html
 <script>
-  // Check for existence of Sentry in case Ad-blockers block the Sentry Loader Script
-  window.Sentry && Sentry.onLoad(function() {
-    Sentry.init({
-      release: " ... ",
-      environment: " ... "
-    });
-    Sentry.configureScope(scope => {
-      scope.setTag( ... );
-    });
-    // etc.
-  });
+  // Configure sentryOnLoad before adding the Loader Script
+    window.sentryOnLoad = function () {
+      Sentry.init({
+        release: " ... ",
+        environment: " ... "
+      });
+      Sentry.setTag(...);
+      // etc.
+    };
 </script>
 ```
 
-For Performance Monitoring, by default, the SDK will be initialized with `tracesSampleRate: 1`. This means that the SDK will capture all traces.
-
-For Session Replay, the defaults are set to `replaysSessionSampleRate: 0.1` and `replaysOnErrorSampleRate: 1`. This means Replays will be captured for 10% of all normal sessions and for all sessions with an error.
+In addition to `window.sentryOnLoad`, you can also use `Sentry.onLoad()` to make sure the SDK is loaded before you call any other SDK methods. The loader will by default make sure that these functions can be called directly on `Sentry` at all times:
```

Comment:
Hmm I'm wondering if we should even still show this. Is there a concrete advantage users gain from doing it the old way? If not wdyt about deprecating the function and
a) adding a deprecation note to this section
b) entirely removing the function from the docs?

No strong opinions here though so logaf-l for me

Reply:
So `onLoad` is only useful now for guarding for the SDK to be loaded - so this should _not_ be used to call `Sentry.init()` anymore, but only if you want to call any other SDK method. I tried to make this clear but apparently it is not yet so :D I'll tweak it a bit to hopefully clarify this!

Reply:
I actually moved the `onLoad()` stuff into a dedicated new section "Guarding SDK Function Calls" to make this clearer! does that make sense to you?